test/testing.sh

```diff
@@ -485,6 +487,9 @@ fail_unpack_invalidrockspec() { need_luasocket; $luarocks unpack "invalid.rocksp
 
 fail_upload_invalidrockspec() { $luarocks upload "invalid.rockspec"; }
 fail_upload_invalidkey() { $luarocks upload --api-key="invalid" "invalid.rockspec"; }
+fail_upload_skippack() { $luarocks upload --api-key="invalid" --skip-pack "luacov-${verrev_luacov}.rockspec"; }
```

Comment:
These tests fail because the rockspec doesn't exist, is this intended? There is a test for a missing rockspec already.